tests/pages/donate.py

```diff
@@ -0,0 +1,10 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.page import Page
+
+
+class DonatePage(Page):
+
+    _url = 'https://donate.mozilla.org/'
```

Comment:
I think seeing as this is an external page let's put this URL in the test and avoid having this file here? Perhaps add a comment to the test for clarity?